sdk/python/feast/infra/utils/snowflake_utils.py

```diff
@@ -429,3 +433,176 @@ def parse_private_key_path(key_path: str, private_key_passphrase: str) -> bytes:
     )
 
     return pkb
+
+
+def write_pandas_binary(
```

Comment:
why is this cloned from https://github.com/snowflakedb/snowflake-connector-python/blob/main/src/snowflake/connector/pandas_tools.py#L38 ? Can you leave comments any time this happens + open an issue to resolve it?

Reply:
write_pandas() writes python binary to snowflake variant. This function modifies it to write python binary to snowflake binary -- I have put in the request to make this a support function in the other repo

will add comments